### ISSUE TITLE
vatz-plugin-sysutil never send SUCCESS message

### ISSUE THREAD

**Issue by xellos00** [dsrvlabs/vatz#336]
### Checklist
- [ ] Bug from Service (Vatz)
- [x] Bug from Plugin (Vatz) 
- [ ] Malfunction on CI/CD 
- [ ] etc(e.g, Hardware version doesn't match, )

---

### **Describe the bug**
> A clear and concise description of what the bug is. There's no Success flags, 
<img width="650" alt="image" src="https://user-images.githubusercontent.com/6308023/206830162-dd6f19b9-d883-45c1-a1c0-4995dd42a47c.png">
<img width="671" alt="image" src="https://user-images.githubusercontent.com/6308023/206830171-18a9c0ec-0ecc-45d2-846c-70a4264369b8.png">

- CPU  
```
func pluginFeature(info, option map[string]*structpb.Value) (sdk.CallResponse, error) {
	// TODO: Fill here.
	ret := sdk.CallResponse{
		FuncName:	"getCPUUsage",
		Message:	"CPU usage warning!",
		Severity:	pluginpb.SEVERITY_UNKNOWN,
		State:		pluginpb.STATE_NONE,
		AlertTypes:	[]pluginpb.ALERT_TYPE{pluginpb.ALERT_TYPE_DISCORD},
	}

	hostname, err := os.Hostname()
	if err != nil {
		log.Fatal().Str("module", "plugin").Msgf("Couldn't get hostname: %v", err)
	}

	util, _ := cpu.Percent(defaultDuration * time.Second, false)
	log.Debug().Str("module", "plugin").Int("CPU Usage", int(util[0])).Int("Urgent", urgent).Int("Warning", warning).Msg("cpu_monitor")

	if int(util[0]) > urgent {
		var message string
		message = fmt.Sprint("[", hostname, "]\n", "Current CPU Usage is ", int(util[0]), "%, over urgent threshold ", urgent, "%")
		ret = sdk.CallResponse{
			FuncName:	"getCPUUsage",
			Message:	message,
			Severity:	pluginpb.SEVERITY_CRITICAL,
			State:		pluginpb.STATE_FAILURE,
			AlertTypes:	[]pluginpb.ALERT_TYPE{pluginpb.ALERT_TYPE_DISCORD},
		}

		log.Warn().Str("module", "plugin").Msg(message)
	}

	return ret, nil
}
```

- Memory
```
func pluginFeature(info, option map[string]*structpb.Value) (sdk.CallResponse, error) {
	// TODO: Fill here.
	ret := sdk.CallResponse{
		FuncName:	"getMEMUsage",
		Message:	"Memory usage warning!",
		Severity:	pluginpb.SEVERITY_UNKNOWN,
		State:		pluginpb.STATE_NONE,
		AlertTypes:	[]pluginpb.ALERT_TYPE{pluginpb.ALERT_TYPE_DISCORD},
	}

	hostname, err := os.Hostname()
	if err != nil {
		log.Fatal().Str("module", "plugin").Msgf("Couldn't get hostname: %v", err)
	}

	v, _ := mem.VirtualMemory()
	log.Debug().Str("module", "plugin").Int("Memory Usage", int(v.UsedPercent)).Int("Urgent", urgent).Int("Warning", warning).Msg("mem_monitor")

	if int(v.UsedPercent) > urgent {
		var message string
		message = fmt.Sprint("[", hostname, "]\n", "Current Memory Usage is ", int(v.UsedPercent), "%, over urgent threshold ", urgent, "%")
		ret = sdk.CallResponse{
			FuncName:	"getMEMUsage",
			Message:	message,
			Severity:	pluginpb.SEVERITY_CRITICAL,
			State:		pluginpb.STATE_FAILURE,
			AlertTypes:	[]pluginpb.ALERT_TYPE{pluginpb.ALERT_TYPE_DISCORD},
		}

		log.Warn().Str("module", "plugin").Msg(message)
	}

	return ret, nil
}
```

- Disk
```
func pluginFeature(info, option map[string]*structpb.Value) (sdk.CallResponse, error) {
	// TODO: Fill here.
	ret := sdk.CallResponse{
		FuncName:	"getDISKUsage",
		Message:	"Disk usage warning!",
		Severity:	pluginpb.SEVERITY_UNKNOWN,
		State:		pluginpb.STATE_NONE,
		AlertTypes:	[]pluginpb.ALERT_TYPE{pluginpb.ALERT_TYPE_DISCORD},
	}

	hostname, err := os.Hostname()
	if err != nil {
		log.Fatal().Str("module", "plugin").Msgf("Couldn't get hostname: %v", err)
	}

	used := make([]int, len(mountPaths))
	for i := 0; i < len(mountPaths); i++ {
		temp, _ := disk.Usage(mountPaths[i])
		used[i] = int(temp.UsedPercent)
		log.Debug().Str("module", "plugin").Int(mountPaths[i], used[i]).Int("Urgent", urgent).Int("Warning", warning).Msg("disk_monitor: Disk Usage(%) of")
		if used[i] > urgent {
			var message string
			message = fmt.Sprint("[", hostname, "]\n", "Current Disk Usage of ", mountPaths[i], used[i], "%, over urgent threshold ", urgent, "%")
			ret = sdk.CallResponse{
				FuncName:	"getDISKUsgae",
				Message:	message,
				Severity:	pluginpb.SEVERITY_CRITICAL,
				State:		pluginpb.STATE_FAILURE,
				AlertTypes:	[]pluginpb.ALERT_TYPE{pluginpb.ALERT_TYPE_DISCORD},
			}

			log.Warn().Str("module", "plugin").Msg(message)
		}
	}

	return ret, nil
}
```

### **How to Reproduce**
> Steps to reproduce the behavior:

1. Start VATZ with plugin is running. 
2. Stop plugin. 
3. sometime if send two error message that 
- CRITICAL plugin is down from healthchecker
- ERROR plugin is dispatcher
4. Restart plugin
5. Healthcheck will be recovered, 
but dispatcher doesn't recover due to it's flags. 

----

### **Expected behavior**
> A clear and concise description of what you expected to happen.

See recovery on both. 
because our Stop sending a reminder flag is 
- STATE_SUCCESS
- SEVERITY_INFO

---

**Comment by xellos00** [comment]
@heejin-github 
I think you need to take a look on this issue because, It never going to stop reminder even if node status is reverted back from abnormal to normal.

**Comment by heejin-github** [comment]
> @heejin-github I think you need to take a look on this issue because, It never going to stop reminder even if node status is reverted back from abnormal to normal.
> 
> <img alt="image" width="830" src="https://user-images.githubusercontent.com/6308023/206830568-d64c58c7-d676-4b01-a823-41dd616b0287.png">

@xellos00 Thanks for letting me know. I'll fix this issue on #327 
- Duplicated : #327

**Comment by xellos00** [comment]
> > @heejin-github I think you need to take a look on this issue because, It never going to stop reminder even if node status is reverted back from abnormal to normal.
> > <img alt="image" width="830" src="https://user-images.githubusercontent.com/6308023/206830568-d64c58c7-d676-4b01-a823-41dd616b0287.png">
> 
> @xellos00 Thanks for letting me know. I'll fix this issue on #327
> 
> * Duplicated : [Sysutil - plugin - cannot recover from alert status #327](https://github.com/dsrvlabs/vatz/issues/327)

I will close this issue because it's duplicated.
